### PR TITLE
fixed some grammer mistakes in docs

### PR DESCRIPTION
**Pull request by LOLdevelopr**
### Description
Fixing of Grammatical mistakes.

### Affected components 

- Server
- Console
- CLI
- [X] Docs
- Community Content
- Build System
- Tests
- Other (list it)

### Related Issues

NA

### Solution and Design

NA

### Steps to test and verify

NA

### Limitations, known bugs & workarounds

NA

**Note**
Sorry for those NA, I couldnt think of any content for this changes in them. Please suggest me if you found any !

### CODE REVIEW

File: docs/graphql/manual/deployment/logging.rst [hunk 1]
```diff
@@ -20,9 +20,9 @@ Based on your deployment method, the Hasura GraphQL engine logs can be accessed
 Different log-types
 -------------------
 
-The Hasura GraphQL engine has different kind of log-types depending on the sub-system or the layer. A log-type is simply the ``type`` field in a log line, which indicates which sub-system the log comes from.
+The Hasura GraphQL engine has a different kind of log-types depending on the sub-system or the layer. A log-type is simply the ``type`` field in a logline, which indicates which sub-system the log comes from.
```
Comment (marionschleifer) [inline docs/graphql/manual/deployment/logging.rst]: That is not correct. Please undo this change.

File: docs/graphql/manual/queries/custom-functions.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ Query custom SQL Functions
   :depth: 2
   :local:
 
-What are custom SQL functions?
+What are the custom SQL functions?
```
Comment (marionschleifer) [inline docs/graphql/manual/queries/custom-functions.rst]: That is not correct. Please undo this change.

**Comment by LOLdevelopr** [comment]
@marionschleifer I changed those which you mentioned

**Comment by marionschleifer** [comment]
@LOLdevelopr could you please send an email to marion@hasura.io, so that we can arrange the shipping of your swag? 🙂

**Comment by LOLdevelopr** [comment]
Done 👍
